regression-tests/mixed-lifetime-safety-pointer-init-1-error.cpp2

```diff
@@ -1,13 +1,14 @@
 
-#include <cstdlib>
 #include <iostream>
+#include <random>
 
 main: () -> int = {
     x: int = 42;
     p: *int;
 
     // ... more code ...
-    if std::rand()%2 {
+    // the standard mandates that std::mt19937()() == 3499211612
+    if std::mt19937()()%2 {
```

Comment:
[Question] What do you think about disallowing implicit conversions of integer types to `bool`?

Reply:
I plan to add that sometime down the road, yes.